docs/index.md

```diff
@@ -55,7 +58,7 @@ By default, SDKs need to be configured with a DSN that points to `sentry.io`. If
 you need to restrict all HTTP communication to a custom domain name, Relay can
 act as an opaque proxy that reliably forwards events to Sentry.
 
-## Getting Started
+# Getting Started
```

Comment:
Let's make this second level again, as it should be lower than the top-level heading.

```suggestion
## Getting Started
```